src/include/sof/audio/codec_adapter/interfaces.h

```diff
@@ -15,16 +15,11 @@
 #include <sof/audio/codec_adapter/codec/cadence.h>
 #endif /* CONFIG_CADENCE_CODEC */
 
-#if CONFIG_DUMMY_CODEC
-#include <sof/audio/codec_adapter/codec/dummy.h>
-#endif
-
 #if CONFIG_WAVES_CODEC
 #include <sof/audio/codec_adapter/codec/waves.h>
 #endif
 
 #define CADENCE_ID 0xCADE01
-#define DUMMY_ID   0xD03311
```

Comment:
shouldn't we also get rid of CADENCE_ID and WAVES_ID ?

Reply:
@mrajwa will do this but gradually. This commit only deals with ~~Dummy~~ passthrough codec.